src/app/app.component.html

```diff
@@ -1,5 +1,7 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  Your Stl Courts Page
+<div fxLayout="column" fxFlexFill>
+  <app-header></app-header>
+  <div fxFlex fxLayout="column">
+    <router-outlet hidden-router></router-outlet>
+  </div>
+  <app-footer></app-footer>
```

Comment:
Is this layout providing a sticky footer like the existing site? This means the footer is always stuck to bottom of the window regardless of how long the document (i.e. rendered content) is. Otherwise there is empty space below the footer.

Reply:
Yes, see comment above.